### PR TITLE
Add PHP 8.4 to CI

### PR DESCRIPTION
**Pull request by andreybolonin**
|      Q       |   A
|------------- | -----------
| Type         | bug/feature/improvement
| Fixed issues |

### CODE REVIEW

**Comment by greg0ire** [comment]
:thinking: isn't this a bit premature?

**Comment by derrabus** [comment]
> 🤔 isn't this a bit premature?

I don't think so. We can start running the CI against PHP 8.4, but if we do, we should test all extensions and not just SQLite.

**Comment by andreybolonin** [comment]
@derrabus updated for all extentions

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -299,6 +302,9 @@ jobs:
           - php-version: "8.3"
             mariadb-version: "11.2"
             extension: "pdo_mysql"
+          - php-version: "8.4"
+            mariadb-version: "11.2"
+            extension: "pdo_mysql"
```
Comment (derrabus) [inline .github/workflows/continuous-integration.yml:307]: We need to test with both extensions!

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -370,6 +376,9 @@ jobs:
           - php-version: "8.3"
             mysql-version: "8.0"
             extension: "pdo_mysql"
+          - php-version: "8.4"
+            mysql-version: "8.0"
+            extension: "pdo_mysql"
```
Comment (derrabus) [inline .github/workflows/continuous-integration.yml:381]: Same here

**Review by derrabus — CHANGES_REQUESTED** [review]

**Comment by derrabus** [comment]
And please target 3.8.x for this PR.

**Comment by derrabus** [comment]
🤷🏻‍♂️

**Pull request closed without merging**